wasm.js

```diff
@@ -0,0 +1,185 @@
+/*
+ * This file is part of the 0chain @zerochain/0chain distribution (https://github.com/0chain/client-sdk).
+ * Copyright (c) 2018 0chain LLC.
+ *
+ * 0chain @zerochain/0chain program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+'use strict';
+
+const utils = require('./utils');
+
+// Initially, the __zcn_wasm__ object will be
+const g = global || window || self;
+
+/**
+ * bridge is an easier way to refer to the Go WASM object.
+ */
+const bridge = g.__zcn_wasm__;
+
+async function blsSign (hash) {
+  if (!bridge.bls?.secretKey) {
+    const msg = 'err: bls.secretKey is not initialized';
+    console.error(msg);
+    return msg;
+  }
+
+  try {
+    const bytes = utils.hexStringToByte(hash);
+
+    const sig = bridge.bls.secretKey.sign(bytes);
+
+    if (!sig) {
+      const msg = 'err: failed to sign';
+      return msg;
+    }
+
+    return sig.serializeToHexStr();
+  } catch (error) {
+    console.error(error);
+    return 'err: ' + error.Error();
+  }
+}
+
+if (!g.__zcn_wasm__) {
+	g.__zcn_wasm__ = {
+		bls: {
+			sign: blsSign,
+		},
```

Comment:
It was somewhat confusing having blsSign try to read from bridge.bls.secretKey, until I saw that it refers to it's own parent object. For clarity, we should declare secretKey here, I suggest as null.

```suggestion
		bls: {
		    secretKey: null,
			sign: blsSign,
		},
```